src/test/assets/mediziner-mensa/reference/week_2018_44.json

```diff
@@ -0,0 +1,482 @@
+{
+  "version": "2.1",
+  "canteen_id": "mediziner-mensa",
+  "weeks": [
+    {
+      "number": 44,
+      "year": 2018,
+      "days": [
+        {
+          "date": "2018-10-29",
+          "dishes": [
+            {
+              "name": "Spinatcremesuppe",
+              "prices": {
+                "students": null,
+                "staff": null,
+                "guests": null
+              },
+              "labels": [
+                "Label.GLUTEN",
+                "Label.LACTOSE",
+                "Label.MILK"
+              ],
+              "dish_type": "Suppe"
+            },
+            {
+              "name": "Gekochtes Ochsenfleisch mit Meerrettich",
+              "prices": {
+                "students": null,
+                "staff": null,
+                "guests": null
+              },
+              "labels": [
+                "Label.ANTIOXIDANTS",
```

Comment:
I don't think this is expected output here, right?

Reply:
Has been like this before. But I'll add the price to staff and guests now, too. According to the [testfile](https://github.com/TUM-Dev/eat-api/blob/495c1530c8590a9edb5518782632eb6bb2f9a052/src/test/assets/mediziner-mensa/for-generation/week_2018_44.txt) this should be fine. There does not seem to be a special price for staff or guests.  
However, I can't verify whether this is the case now, since I can't find the canteen's menu on there webpage. Probably the canteen is closed.

Reply:
I just added a new issue for the problem. See #67.